README.md

```
@@ -140,14 +86,14 @@ Figma에서 포게더의 자세한 디자인 시스템을 확인할 수 있어
             <a href="https://github.com/H0ngJu"><I>H0ngJu</I></a>
         </td>
         <td>
-            <img src="https://avatars.githubusercontent.com/u/80245376?v=4?size=100" width="100">
+            <img src="https://avatars.githubusercontent.com/u/124787447?v=4?size=100" width="100">
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)